save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/OrganizationController.kt

```diff
@@ -114,6 +114,23 @@ internal class OrganizationController(
         return Mono.just(response)
     }
 
+    /**
+     * @param organizationName
+     * @param authentication
+     */
+    @DeleteMapping("/{organizationName}/delete")
+    @PreAuthorize("isAuthenticated()")
+    fun deleteOrganization(@PathVariable organizationName: String, authentication: Authentication): Mono<StringResponse> {
+        val role = lnkUserOrganizationService.getGlobalRoleOrOrganizationRole(authentication, organizationName)
+        val response = if (role.priority >= Role.OWNER.priority) {
+            organizationService.deleteOrganization(organizationName)
+            ResponseEntity.ok("Organization deleted")
+        } else {
+            ResponseEntity.status(HttpStatus.FORBIDDEN).build()
+        }
+        return Mono.just(response)
```

Comment:
btw - please unify  the body of the function into a single Mono (functionally with `map`)

Reply:
ok, will do it